.github/workflows/pr.yaml

```
@@ -0,0 +1,127 @@
+on: pull_request
+name: PR Review
+jobs:
+  changelog:
+    name: Changelog should be updated
+    runs-on: ubuntu-20.04
+    strategy:
+      fail-fast: false
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+
+      - name: Git fetch
+        run: git fetch
+
+      - name: Check that changelog has been updated.
+        run: git diff --exit-code origin/develop -- CHANGELOG.md && exit 1 || exit 0
```

`develop` is not always the base of a pull request:

```suggestion
        run: git diff --exit-code origin/${{ github.base_ref }} -- CHANGELOG.md && exit 1 || exit 0
```